### ISSUE TITLE
In experiment II and experiment I both are different from yours.

### ISSUE THREAD

**Issue by litingting1111** [sibrendebast/inside-the-black-box#2]
Experiment I results shows the positioning accuracy in the nLoS is better than in the LoS and Experiment  II, the reappearance result is different from yours.
![cdf_positioning](https://user-images.githubusercontent.com/80535260/220329704-242a3a21-6b3b-44e2-a719-c2be36f2b1ef.png)
![moving_comparison_URA_lab_LoS](https://user-images.githubusercontent.com/80535260/220329714-74686e69-c921-4fce-89f3-13155331f3d5.png)

**Comment by litingting1111** [comment]
Maybe you can give some training details. including the epoch and batch size. My experiment follow your default.

**Comment by sibrendebast** [comment]
Hi,

I assume you are using the dataset found on https://ieee-dataport.org/open-access/ultra-dense-indoor-mamimo-csi-dataset

This dataset was made after this code and the corresponding paper were published. There are some minor differences in the dataset that can explain these differences. 

For the first experiment, I also saw in more recent results that the positioning performance is better in the case of the nLoS scenario, however, this is only the case when a lot of training data is used. If you would reduce the amount of training data, the LoS scenario has a higher performance.

For experiment II, You should check if the order of the dataset is the same. I assume that the specific order in which the nomadic scenario is stored in which array might have changed. I refer to the documentation of the dataset for this and to the code for experiment II.

I hope this answer was helpful.

**Comment by litingting1111** [comment]
thanks, i guess i can get it.

**Comment by msh003** [comment]
> Experiment I results shows the positioning accuracy in the nLoS is better than in the LoS and Experiment II, the reappearance result is different from yours. ![cdf_positioning](https://user-images.githubusercontent.com/80535260/220329704-242a3a21-6b3b-44e2-a719-c2be36f2b1ef.png) ![moving_comparison_URA_lab_LoS](https://user-images.githubusercontent.com/80535260/220329714-74686e69-c921-4fce-89f3-13155331f3d5.png)


Hello, where did you find labels.npy file in the data set?
